rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -22,6 +22,7 @@
 import amd.rocal.types as types
 import numpy as np
 import cupy as cp
+import torch
```

Comment:
@SundarRajan28 why  are we brining this back? This would make  torch as a pre req for all use cases. That's why we moved to cupy

Reply:
Removed the torch import and usage of torch APIs in the latest commit